config/routes.rb

```diff
@@ -1,6 +1,10 @@
 FloodRiskEngine::Engine.routes.draw do
+  resources :enrollments
+
+  # Copied from dummy app - may be out of date now
   resources :enrollments, only: [:new] do
```

Comment:
The enrollment resources should only be defined once. Rather than adding an additional `resources :enrollments` the existing one should be modified to added the required additional actions. So:

```
resources :enrollments, only: [:new, :create] do
  resources :steps, only: [:show, :update], controller: "enrollments/steps"
end
```